src/main/java/org/ibp/api/java/impl/middleware/germplasm/GermplasmServiceImpl.java

```diff
@@ -337,8 +337,10 @@ public long countAttributesByGid(final String gid, final List<String> attributeD
 		this.germplasmUpdateValidator.validateAttributeAndNameCodes(this.errors, germplasmUpdateDTOList);
```

Comment:
I think we should abstract all this validations inside `GermplasmUpdateValidator`, which we should update name to `GermplasmUpdateDtoValidator` (similar to `germplasmImportRequestDtoValidator.validate`)